packages/vue-language-core/src/utils/directorySharedTypes.ts

```diff
@@ -17,13 +17,31 @@ type __VLS_Prettify<T> = {
 	[K in keyof T]: T[K];
 } & {};
 
+// @ts-ignore
+type __VLS_VueGlobalComponents = import('vue').GlobalComponents;
+// @ts-ignore
+type __VLS_VueRuntimeCoreGlobalComponents = import('@vue/runtime-core').GlobalComponents;
+// @ts-ignore
+type __VLS_VueRuntimeDomGlobalComponents = import('@vue/runtime-dom').GlobalComponents;
+
+// Hack: \`Type & Type\` is a hack to make unresolved imports' type to be "real any" in TS 5.1.3.
+// Since typescript 5.1.3, the behavior of unresolved imports has changed.
+// For example, the following code:
+// \`\`\`ts
+// type A = import('foo').NotExist extends any ? true : false;
+// type B = A extends any ? true : false;
+// \`\`\`
+// In typescript 5.0.4, A and B are both \`true\`.
+// However, in 5.1.3, A becomes \`any\` and B becomes \`any\` too!
+// Actually, any type which includes unresolved imports will be turned into \`any\`.
+// Even only a reference, like B.
+// So we need to turn it into a \`real any\` and avoid pollution.
+// This hack does it by intersect itself with itself 
+// Ref: #3274
```

Comment:
Why are you escaping characters within the comment?

Reply:
Because I use backsticks to wrap code snippets, but the template literal also uses backsticks

Reply:
This is just a plain comment, not parsed as markdown by anything. So I don't see the point of escaping. Just use a plain text.